dynamo-event-to-datacite-request/src/main/java/no/unit/nva/events/models/AwsEventBridgeEvent.java

```diff
@@ -39,12 +39,12 @@ public AwsEventBridgeEvent() {
     @JacocoGenerated
     @JsonProperty("region")
     public String getRegion() {
-        return region.getName();
+        return Optional.ofNullable(region).map(Regions::getName).orElse(null);
     }
 
     @JacocoGenerated
     public void setRegion(String region) {
-        this.region = Regions.fromName(region);
+        this.region = Optional.ofNullable(region).map(Regions::fromName).orElse(null);
```

Comment:
Why not make return type `Optional<String>` ?
or is it totally fine that this is `null` for the caller?

Reply:
this is a setter. I am not sure if it is a good idea to have an Optional field.

Reply:
Oops, read the code a bit too fast.